census/vanilla-tf/trainer/task.py

```diff
@@ -0,0 +1,338 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""This code implements a Feed forward neural network using TF low level APIs.
+   It implements a binary classifier for Census Income Dataset using both single
+   and distributed node cluster. 
+"""
+
+
+import argparse
+import json
+import tensorflow as tf
+import numpy as np
+import pandas as pd
+from tensorflow.contrib.layers.python.ops import sparse_feature_cross_op
+from tensorflow.contrib.layers.python.ops import bucketization_op
+
+from StringIO import StringIO
+
+import model
+import os
+
+tf.logging.set_verbosity(tf.logging.INFO)
+
+#See tutorial on wide and deep https://www.tensorflow.org/tutorials/wide_and_deep/
+#https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/layers/python/layers/feature_column.py
+
+# csv columns in the input file
+CSV_COLUMNS = ('age', 'workclass', 'fnlwgt', 'education', 'education_num',
+               'marital_status', 'occupation', 'relationship', 'race', 'gender',
+               'capital_gain', 'capital_loss', 'hours_per_week', 'native_country',
+               'income_bracket')
+
+CATEGORICAL_COLS = ('gender', 'race', 'education', 'marital_status',
+                    'relationship', 'workclass','occupation', 'native_country')
+
+CONTINUOUS_COLS = ('age', 'education_num', 'capital_gain', 'capital_loss', 'hours_per_week')
+
+LABEL_COL = 'income_bracket'
+
+
+#
+# Graph creation section for training and evaluation
+#
+def make_graph(inputs, labels, learning_rate=0.5):
```

Comment:
So I am already producing train and eval step the way you describe. Prediction not because this sample doesn't do prediction. Do you expect prediction as well in this one?